src/controller/batches/batch.service.ts

```diff
@@ -21,6 +21,16 @@ export class BatchesService {
           sql`${zuvyBatchEnrollments.bootcampId} = ${batch.bootcampId} AND ${zuvyBatchEnrollments.batchId} IS NULL`,
         )
         .limit(batch.capEnrollment);
+      const instructorRoles = await db.select({role:sansaarUserRoles.role}).from(sansaarUserRoles).where(eq(sansaarUserRoles.userId,batch.instructorId)) 
```

Comment:
Please ensure that the instructor is effectively onboarded.

Reply:
Maybe we don't need this condition. let's connect in the mrng we can do those change. for now let it be.